bin/playwright.sh

```diff
@@ -9,8 +9,8 @@ export PLAYWRIGHT_ARGS=$@
 export PLAYWRIGHT_VERSION=$(version)
 export TEST_COMMAND=${TEST_COMMAND:-test:playwright:local}
 
-echo Running Playwright v$PLAYWRIGHT_VERSION as $USER_ID with Playwright arguments $PLAYWRIGHT_ARGS
+cp -r test/locales/* src/locales
 
-docker-compose -f docker-compose.playwright.yml up --build --force-recreate --abort-on-container-exit --exit-code-from playwright --remove-orphans
+echo Running Playwright v$PLAYWRIGHT_VERSION as $USER_ID with Playwright arguments $PLAYWRIGHT_ARGS
 
-docker-compose -f docker-compose.playwright.yml down
+docker-compose -f docker-compose.playwright.yml up --build --force-recreate --exit-code-from playwright --remove-orphans
```

Comment:
So was it the `docker-compose down` that was causing this not to fail because the script continued to the `dc down` command after the failure of the test run? If so, an alternative fix (assuming the `down` is necessary) would be to add `set -x` to the top of the script so that any command failure causes the script to exit with the failure exit code.

Reply:
Well, if you remove the `dc down` line, the tests start to fail again. But I'm not very clear as to what exactly happens.

Reply:
If that is the case then I think it is what I said. The default behavior with shell scripts is not to bail out the entire script if a command fails unless the `e` option is set. 

I had remembered it wrong, btw, it should be `set -e`. `set -x` prints the commands as they're run. `e` bails on the script if any error occurs.

Reply:
It worked! Thank you :)
How did you see that the Docker container was still running after tests in GitHub actions, by the way?

Reply:
>  How did you see that the Docker container was still running after tests in GitHub actions, by the way?

What do you mean?

Reply:
I was going to suggest using `docker-compose run --rm ...` instead but glad to see it's working now! :)

Reply:
I don't remember why but `dc run` did not work for me when I tried it when originally developing the dockerized Playwright tests. There was something strange about it where it would never end the containers, IIRC.

Reply:
> What do you mean?

The `dc down` line was added to solve the problem where a Playwright container wouldn't stop, right? I have never known that it didn't stop in the CI. How did you understand that that **was**, indeed, happening?

Reply:
I think @krysal added it (IIRC) upon noticing that the containers weren't stopping _locally_. I don't think it matters if the containers continue to run in CI, as far as I know.

Reply:
> I think @krysal added it (IIRC) upon noticing that the containers weren't stopping locally.

That is correct. The machines running the GH actions are a big black box, so that would have been much more difficult to identify, though locally it was a coincidence 🙂